Makes/ViewController.m

```diff
@@ -0,0 +1,146 @@
+//
+//  ViewController.m
+//  Makes
+//
+//  Created by Mohamed Ragab on 1/8/17.
+//  Copyright © 2017 Mohamed Ragab. All rights reserved.
+//
+
+#import "ViewController.h"
+#import "Make.h"
+#import "AFNetworking.h"
+#import "Masonry.h"
+#import "MakesCollectionViewCell.h"
+#import "MakeDetailViewController.h"
+
+
+
+
+@interface ViewController ()
+@property (nonatomic, strong) NSMutableArray *makes;
+@property (nonatomic,strong) NSCache* imagesCache;
+@end
+
+@implementation ViewController
+
+- (void)loadMakes
+{
+    AFHTTPSessionManager *manager = [AFHTTPSessionManager manager];
+    
+    NSString *path = [[NSBundle mainBundle] pathForResource:@"url_string_file" ofType:@"txt"];
+    NSString *content = [NSString stringWithContentsOfFile:path encoding:NSUTF8StringEncoding error:nil];
+
+    
+    [manager GET:content parameters:nil progress:nil success:^(NSURLSessionTask *task, id responseObject) {
+        
+        NSArray* dictionariesArray = (NSArray*)responseObject;
+        NSLog(@"%@",responseObject);
```

Comment:
whenever you make sure that something is working fine, stop logging it (remove this line)
also you can use breakpoints to debug.
http://jeffreysambells.com/2014/01/14/using-breakpoints-in-xcode